### PR TITLE
Optimize the rate limiter parameters

### PR DESCRIPTION
**Pull request by luobily**
#### What type of this PR

Add one of the following kinds:
/kind polish

#### What this PR does / why we need it:
- Optimize the rate limiter parameters.
- Adjust the volume provisioner name to dice prefix, otherwise it will cause the existing cluster upgrade to fail
- Optimize the rolling upgrade ratio to 25%

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @iutx @sixther-dc

### CODE REVIEW

File: modules/volume-provisioner/deploy/pvc.yaml [hunk 1]
```diff
@@ -8,4 +8,4 @@ spec:
   resources:
     requests:
       storage: 8Gi
-  storageClassName: erda/local-volume
+  storageClassName: dice/local-volume
```
Comment (iutx) [inline modules/volume-provisioner/deploy/pvc.yaml:11]: dice/local-volume -> dice-local-volume
kubernetes storageClassName can't use "/" here.

**Review by iutx — APPROVED** [review]
/lgtm

**Comment by luobily** [comment]
cc @Effet

**Comment by sfwn** [comment]
Your commits must be signed.

**Pull request merged**